packages/shared/src/components/filters/MyFeedHeading.tsx

```diff
@@ -36,6 +42,8 @@ function MyFeedHeading({
   const isLaptop = useViewSize(ViewSize.Laptop);
   const feedName = getFeedName(router.pathname);
   const { isCustomFeed } = useFeedName({ feedName });
+  const shortcutsUIFeature = useFeature(feature.shortcutsUI);
+  const isShortcutsUIV1 = shortcutsUIFeature === ShortcutsUIExperiment.V1;
```

Comment:
This will do enrollment for everyone even if they not on extension even right?

Reply:
yes, good catch

Reply:
3633f571c80d81f2fbb51f1c7fc67e171ff0b0fc